packages/google_maps_flutter/google_maps_flutter/android/src/main/java/io/flutter/plugins/googlemaps/Convert.java

```diff
@@ -197,6 +205,18 @@ static Object circleIdToJson(String circleId) {
     return data;
   }
 
+  static Object tileOverlayArgumentsToJson(String tileOverlayId, int x, int y, int zoom) {
```

Comment:
Return type can be ` Map<String, Object>`

Reply:
Yes, the type may be like that. In this class, the other methods all return an Object. Think not to pay attention to other methods and return the Map?

Reply:
The same is in your other comments. I did in the same style as the existing code

Reply:
I'm not sure why it was like that in other places in the code. Probably some historical reasons.
I would follow the best practice (to indicate the actual return type instead of always use a generic Object) in the new code.

Do you mind doing it? It would help me reviewing the PR much easier :)